katportalclient/client.py

```diff
@@ -407,10 +527,11 @@ def _websocket_message(self, msg):
         if msg is None:
             self._logger.warn("Websocket server disconnected!")
             if not self._disconnect_issued:
-                self._ws.close()
-                self._ws = None
+                if self._ws is not None:
+                    self._ws.close()
+                    self._ws = None
                 yield self._connect(reconnecting=True)
-            return
+            raise tornado.gen.Return()
```

Comment:
Why did you make this change? It is not needed/is no different. 
And personally I find a return prettier. You dont have to change anything, I am just curious.

From Tornado Src:
> Coroutines may "return" by raising the special exception
>     `Return(value) <Return>`.  In Python 3.3+, it is also possible for
>     the function to simply use the ``return value`` statement (prior to
>     Python 3.3 generators were not allowed to also return values).
>     In all versions of Python a coroutine that simply wishes to exit
>     early may use the ``return`` statement without a value.
and after some more bla bla bla bla
> By analogy with the return statement, the value argument is optional,
> but it is never necessary to ``raise gen.Return()``.  The ``return``
> statement can be used with no arguments instead.